frontend/dockerfile/dockerfile2llb/convert_test.go

```diff
@@ -41,4 +41,12 @@ COPY --from=0 f2 /
 		Target: "nosuch",
 	})
 	assert.Error(t, err)
+
+	df = `FROM "" AS foo`
+	_, _, err = Dockerfile2LLB(appcontext.Context(), []byte(df), ConvertOpt{})
+	assert.Error(t, err)
```

Comment:
q: What was the behavior/error before that change in buildkit?

Reply:
These errors occur before that change. Sorry I misunderstand buildkit already used in docker.

```
--- FAIL: TestDockerfileParsing (6.87s)
        Error Trace:    convert_test.go:47
	Error:		Received unexpected error invalid reference format

        Error Trace:    convert_test.go:51
	Error:		Received unexpected error invalid reference format

FAIL
FAIL	github.com/moby/buildkit/frontend/dockerfile/dockerfile2llb	6.896s
```